packages/fluentui/docs/src/components/MarkdownPage.tsx

```diff
@@ -19,27 +19,45 @@ type MarkdownPageProps = {
   };
 } & RouteProps;
 
-const components = {
+const components = teamsTheme => ({
   a: ({ children, href }) => link(children, href),
   code: ({ className, children, fitted, label }) =>
     className ? (
       <CodeSnippet fitted={fitted} mode={className.replace('language-', '')} label={label} value={children} />
     ) : (
       <code>{children}</code>
     ),
-  h1: ({ children }) => <Header as="h1" content={children} />,
-  h2: ({ children }) => <Header as="h2" content={children} />,
-  h3: ({ children }) => <Header as="h3" content={children} />,
+  h1: ({ children }) => (
+    <Header
+      as="h1"
+      content={children}
+      styles={({ theme: { siteVariables } }) => ({ color: siteVariables.colors.grey[750] })}
+    />
+  ),
+  h2: ({ children }) => (
+    <Header
+      as="h2"
+      content={children}
+      styles={({ theme: { siteVariables } }) => ({ color: siteVariables.colors.grey[750] })}
+    />
+  ),
+  h3: ({ children }) => (
+    <Header
+      as="h3"
+      content={children}
+      styles={({ theme: { siteVariables } }) => ({ color: siteVariables.colors.grey[750] })}
+    />
+  ),
   img: props => <img style={{ maxWidth: '100%' }} {...props} />,
-};
+});
 
-const MarkdownPage: React.FunctionComponent<MarkdownPageProps> = props => {
+const MarkdownPage: React.FunctionComponent<MarkdownPageProps> = (props, { siteVariables }) => {
   const { page } = props;
   const { default: Component, meta } = page;
 
   return (
     <DocPage title={meta.title}>
-      <MDXProvider components={components}>
+      <MDXProvider components={components(teamsTheme)}>
```

Comment:
```suggestion
      <MDXProvider components={components}>
```

This change can be restored